pyproject.toml

```diff
@@ -13,14 +13,19 @@ exclude = [
     # Files related to the Python autograder will often intentionally appear
     # broken in isolation. We'll exclude these files from linting.
     "exampleCourse/questions/**/tests/initial_code.py",
-    "exampleCourse/questions/**/tests/trailing_code.py"
+    "exampleCourse/questions/**/tests/trailing_code.py",
 ]
 target-version = "py310"
 
 [tool.ruff.lint]
-extend-select = ["I"]
+extend-select = ["I", "N"]
 
 [tool.ruff.lint.per-file-ignores]
+# Only enforce naming convention for our files
+"exampleCourse/**" = ["N806", "N803", "N802", "N813"]
```

Comment:
@reteps if you're still interested in helping with this, the next step is to make these ignore lists as short as possible.

Reply:
I think they are now "as short as possible" :)
https://github.com/eliotwrobson/PrairieLearn/pull/8